src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStatic.Fixer.cs

```diff
@@ -197,23 +201,27 @@ private async Task<(Solution newSolution, bool allReferencesFixed)> UpdateRefere
             // Local functions.
             static bool IsReplacableOperation(IOperation operation)
             {
-                // We only replace reference operations whose removal cannot change semantics.
-                if (operation != null)
+                if (operation == null)
```

Comment:
How do we hit this if the member is not static?

Reply:
I added a test for it - you can access an instance member off a static field/property. Like `SomeType.staticField.IntanceMethod()`

Reply:
Basically, you can reach this path only via the recursive call to `IsReplacableOperation`

Reply:
Ah, I see, maybe add this as a comment?